pkg/webservice/handlers_test.go

```diff
@@ -443,3 +444,108 @@ func prepareSchedulerForConfigChange(t *testing.T) {
 		t.Fatalf("Error when load clusterInfo from config %v", err)
 	}
 }
+
+func TestGetNodesUtilJSON(t *testing.T) {
+	const configDefault = `
```

Comment:
there is already a configDefault defined in TestQueryParamInAppsHandler. Please extract it in a common place and reuse it.

Reply:
done